zerver/webhooks/uptimerobot/doc.md

```diff
@@ -14,6 +14,8 @@ Receive Zulip notifications from UptimeRobot!
     notifications sent to. If you do not specify a `stream`, the bot will
     send notifications via PMs to the creator of the bot.
 
+    {!event-filtering-instruction.md!}
```

Comment:
The [UptimeRobot integration](https://zulip.com/integrations/doc/uptimerobot) seems to have the same structure as the doc for the [Clubhouse integration](
https://zulip.com/integrations/doc/clubhouse). Maybe I'm missing something here, but shouldn't we replace lines 5-17 with `1. {!create-bot-construct-url.md!}`? 

Otherwise, maybe we should remove the `&` from  `{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream=stream%20name&`. The event filtering instructions say "append either `&only_events=["event_a","event_b"]` or `&exclude_events=["event_a","event_b"]`", which already include the `&`.

Reply:
Sounds good! I replaced this part for UptimeRobot with the macro.